lib/rubocop/cop/style/unless_logical_operators.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for the use of logical operators in an unless condition.
```

Comment:
You can add a bit of rationale here - e.g. this cop exists to discourage hard to read/process usages of `unless`.

Reply:
In the rationale you should also briefly explain the purpose of the two supported styles.